common/source/docs/common-auxiliary-functions.rst

```diff
@@ -0,0 +1,633 @@
++----------------------+----------------------------+----------+---------+---------+
+|        59            | SIMPLE mode (Rover)        |          |         |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        60            | ZIGZAG mode                |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        61            | ZIGZAG mode - Save Waypts  |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        62            | Compass Learn              |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        63            | Sailboat Tack              |          |         |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        64            | Reverse Throttle           |          |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        65            | GPS Disable                |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        66            | Relay 5 On/Off             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        67            | Relay 6 On/Off             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        68            | STABILIZE mode             |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        69            | POSHOLD mode               |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        70            | ALTHOLD mode               |    X     |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        71            | FLOWHOLD mode              |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        72            | CIRCLE mode                |    X     |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        73            | DRIFT mode                 |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        74            | Sailboat Motor 3Pos Sw     |          |         |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        75            | Surface Tracking Up/Down   |          |         |    X    |
```

Comment:
Surface Tracking Up/Down is for Copter (not Rover)

Reply:
oops..will change

Reply:
fixed